### PR TITLE
add community post controllers and views

### LINKED ISSUE #47: Community post controllers and views

### PR DESCRIPTION
**Pull request by DavidFonsek**
GG
Agregue community post controllers and views y mil millones de cosas más, gracias a todos por creer en mi
<3

### CODE REVIEW

File: app/Http/Controllers/CommunityPostController.php
Comment (mstermigol) [inline app/Http/Controllers/CommunityPostController.php:1]: Remember to specify the parameter type and return type

Comment (mstermigol) [inline app/Http/Controllers/CommunityPostController.php:1]: Remember to use the setters and getters

Comment (mstermigol) [inline app/Http/Controllers/CommunityPostController.php:1]: Remember the use of lang

Comment (mstermigol) [inline app/Http/Controllers/CommunityPostController.php:1]: Remove comments

File: app/Http/Controllers/TravelController.php
Comment (mstermigol) [inline app/Http/Controllers/TravelController.php:1]: Remove comments

Comment (mstermigol) [inline app/Http/Controllers/TravelController.php:1]: Remember to specify the parameter type and return type

Comment (mstermigol) [inline app/Http/Controllers/TravelController.php:1]: Remember to use getters and setters

Comment (mstermigol) [inline app/Http/Controllers/TravelController.php:1]: Remember to use lang

File: app/Models/CommunityPost.php
Comment (mstermigol) [inline app/Models/CommunityPost.php:1]: Review validation should be in the review model

Comment (mstermigol) [inline app/Models/CommunityPost.php:1]: The name of the method should only be validate. Anything else is redundant

File: app/Models/Travel.php
Comment (mstermigol) [inline app/Models/Travel.php:1]: The name of the method should only be validate. Anything else is redundant

File: resources/views/communitypost/index.blade.php
Comment (mstermigol) [inline resources/views/communitypost/index.blade.php:1]: Use getters and setters

Comment (mstermigol) [inline resources/views/communitypost/index.blade.php:1]: Remove comments. Also remember that everything must be in english

File: resources/views/communitypost/new.blade.php
Comment (mstermigol) [inline resources/views/communitypost/new.blade.php:1]: Remember to use lang

Comment (mstermigol) [inline resources/views/communitypost/new.blade.php:1]: Remember to use getters and setters

File: resources/views/communitypost/show.blade.php
Comment (mstermigol) [inline resources/views/communitypost/show.blade.php:1]: Remember to use getters and setters

Comment (mstermigol) [inline resources/views/communitypost/show.blade.php:1]: Remove comments

File: resources/views/travel/index.blade.php
Comment (mstermigol) [inline resources/views/travel/index.blade.php:1]: Remove comments

Comment (mstermigol) [inline resources/views/travel/index.blade.php:1]: Dont use styles in html. Use the css.app with the "!important"

File: resources/views/travel/show.blade.php
Comment (mstermigol) [inline resources/views/travel/show.blade.php:1]: Remove comment

File: resources/views/communitypost/index.blade.php
Comment (DavidFonsek) [reply resources/views/communitypost/index.blade.php:1]: No

**Pull request merged**